tests/sentry/utils/sdk_crashes/test_sdk_crash_detection.py

```diff
@@ -177,8 +177,15 @@ def test_report_cocoa_sdk_crash_frames(frames, should_be_reported):
     _run_report_test_with_event(event, should_be_reported)
 
 
+def test_sdk_crash_detected_event_is_not_reported():
+    event = get_crash_event()
+    event["type"] = "extracted_sdk_crash"
```

Comment:
I'm not sure what the consequences of changing the event type are. I assume this new event has to go through processing again, which relies on the event type to decide what happens to it. In other words, should we pick another field for this check? As this is a PoC, maybe add a special `Context`?

Reply:
@jjbayer, I agree a different event type might not be the best idea. I'm going to add another marker field somewhere in the context.